### ISSUE TITLE
Performance seems to decrease based on number of calls to chalk

### ISSUE THREAD

**Issue by DeltaF1** [cblgh/sector7557#3]
In areas with high amounts of coloured tiles, the performance drops significantly (doubly so if the tiles have custom bg values). Creating a large amount of tiles with default colours (i.e. after typing `/reset`) does not appear to affect performance.

Platform:
Windows 10 64-bit
Node v10.16.0

**Comment by cblgh** [comment]
yeah i noticed that as well, things can probably be done way smarter when redrawing lol. i'll have a  patching session in the upcoming week or so to try to remedy some of this stuff :)

(it's been toooooo hott these past few days to be inside lol)
